src/main/groovy/com/stormpath/tck/login/LoginIT.groovy

```diff
@@ -116,9 +116,7 @@ class LoginIT extends AbstractIT {
         .then()
             .statusCode(200)
             .contentType(ContentType.JSON)
-            .body("size()", is(2))
```

Comment:
Loosening this up, since my integration doesn't send back account stores in the JSON response anymore. Shouldn't break any existing functionality.